donald_wade/week_04/monday/calculator.rb

```diff
@@ -0,0 +1,101 @@
+# Menu
+# choose from the Menu
+# Enter numbers to perform operation
+# show user the result
+# keep going until user chooses to quit
+
+def multiply(num1, num2)
+  num1 * num2
+end
+
+def divide(num1, num2)
+  num1 / num2
+end
+
+def add(num1, num2)
+  num1 + num2
+end
+
+def subtract(num1, num2)
+  num1 - num2
+end
+
+def power(num1, num2)
+  num1 ** num2
+end
+
+def root(num1)
+  Math.sqrt(num1)
+end
+
+hash_of_symbols = Hash[
+  "*", "multiplication",
+  "/", "division",
+  "+", "addition",
+  "-", "subtraction",
+  "**", "exponent",
+  "sq", "square root",
+]
+
+puts hash_of_symbols
+
+loop do
+  puts 'Which function would you like to perform today?'
+  puts "Enter '*' for multiplication"
+  puts "Enter '/' for division"
+  puts "Enter '+' for addition"
+  puts "Enter '-' for subtraction"
+  puts "Enter '**' for exponent"
+  puts "Enter 'sq' for square root"
+  puts "Enter 'q' to quit"
+
+  action = gets.strip
+  puts action.class
+
+  if action.strip.downcase == "q"
+    break
+  end
+
+  puts "You've chosen to do #{hash_of_symbols[action]}"
+
+  if action != "sq"
+
```

Comment:
no need for specifying != "sq" as you already made a check on it before in your if statement.